c7n/resources/ecs.py

```diff
@@ -392,6 +391,19 @@ def get_resources(self, ids, cache=True):
             self.log.warning("event ids not resolved: %s error:%s" % (ids, e))
             return []
 
+    def augment(self, resources):
+        results = []
+        for task_def_set in resources:
+            client = local_session(self.session_factory).client('ecs')
+            response = client.describe_task_definition(
```

Comment:
look into passing the args into the detail_spec, you should be able to do so without manually doing the describe here, it should be the 3rd position in the detail_spec tuple